### DISCUSSION TITLE
lot of rustdesk outages - icon red or orange - no connections possible - different networks

### DISCUSSION THREAD

**Discussion by aabittner** [rustdesk/rustdesk#8035] [General]
dear Rustdesk team, is it possible to make rustdesk much more resilient and robust? every few days or even hours, there is a lot of outages, rustdesk clients (application) not connecting to your guys backends, relay servers or bootstrap servers or whatnot. the rustdesk status line at its edge of its gui is shown with a red dot or sometimes orange, on windows the relaying service or helper service is okay, or even when shutting down rustdesk and re-starting re-enabling the rustdesk service doesnt help at all.

sometimes it even with green status dot/icon, that the rustdesk app can not connect or after a huge lot of minutes only to remote IDs (peers) etc.

what do you guys need to make this much more robust? thanks a lot for making rustdesk really truly robust and dependable.

**Comment by rustdesk** [comment]
It won't be robust and dependable because it is for demo only. Robustness for millions of users needs a lot of money, we do not have.

# set up your self-hosted server please.
